test/sol5/kyberStorage.js

```diff
@@ -215,19 +258,25 @@ contract('KyberStorage', function(accounts) {
             await kyberStorage.addKyberProxy(proxy1, maxProxies, {from: network});
             await kyberStorage.addKyberProxy(proxy2, maxProxies, {from: network});
 
+            assert(await kyberStorage.isKyberProxyAdded(), "proxy is not added");
+
             await expectRevert(
                 kyberStorage.addKyberProxy(proxy3, maxProxies, {from: network}),
                 "max proxies limit reached"
             );
+
+            proxies = await kyberStorage.getKyberProxies();
+            Helper.assertEqualArray(proxies, [proxy1, proxy2], "unexpected proxies");
         });
 
-        if("test removeKyberProxy revert if not added", async() => {
+        it("test remove proxy revert if not added", async() => {
+            await kyberStorage.addKyberProxy(proxy2, maxProxies, {from: network});
```

Comment:
this is for coverage at https://github.com/KyberNetwork/smart-contracts/blob/bdcf45724c32115ce2d7ce66bb0461be1fd82ab6/contractsSol5/KyberStorage.sol#L183